evento/spec/factories/user_factory.rb

```diff
@@ -0,0 +1,9 @@
+require 'faker'
+
+# This will guess the User class
+FactoryGirl.define do
+  factory :user do
+    name { Faker::Name.name }
```

Comment:
Do you mean `Faker:Name.name`? It just [Faker](https://github.com/stympy/faker) generating "random" name.